ruby-mfa-example/app.rb

```diff
@@ -4,16 +4,12 @@
 require 'sinatra'
 require 'workos'
 require 'json'
-require 'pry'
+
 use Rack::Session::Pool
 
 # Pull API key from ENV variable
 WorkOS.key = ENV['WORKOS_API_KEY']
 
-# Configure your Redirect URIs on the dashboard configuration
-# page: https://dashboard.workos.com/sso/configuration
-
-
 get '/' do
   if session[:factor_list].nil?
     session[:factor_list] ||= []
```

Comment:
When trying to enroll a new SMS factor I got the following error: 
![CleanShot 2022-08-12 at 12 24 00](https://user-images.githubusercontent.com/47123183/184420820-24e60e25-da9d-4362-bf3d-394ae6380987.png)

Reply:
Same with totp

Reply:
It looks like even though we pinned the version, your Ruby version manager is referring to the wrong version.  The updated SDK removes references to environemnt_id (https://github.com/workos/workos-ruby/commit/2a781c9701c0db7e27b6d7890db57701b1000ad9).  I'll mark this as resolved but I'm unsure why you version manager would refer to the wrong workOS version.  I tested this on my other device and also was not able to reproduce